ios/PurchasesHybridCommon/PurchasesHybridCommon/SKProduct+HybridAdditions.m

```diff
@@ -39,59 +45,61 @@ - (NSDictionary *)rc_dictionary
     d[@"introPrice"] = [NSNull null];
     
     if (@available(iOS 11.2, tvOS 11.2, macos 10.13.2, *)) {
-        if (self.introductoryPrice) {
-            d[@"intro_price"] = @(self.introductoryPrice.price.floatValue);
-            d[@"intro_price_string"] = [formatter stringFromNumber:self.introductoryPrice.price];
+        // JOSH: this needs work but has issues because of Decimal being used in swift
+        if (self.introductoryDiscount) {
+            d[@"intro_price"] = @(self.sk1Product.introductoryPrice.price.floatValue);
```

Comment:
Same issue with `Decimal` price on `StoreProduct` does not bridge to Objective-C correctly